gpii/node_modules/flowManager/src/FlowManager.js

```diff
@@ -74,22 +72,41 @@ fluid.defaults("gpii.flowManager", {
         ontologyHandler: {
             type: "gpii.ontologyHandler"
         }
+    }
+});
+
+
+// Mixin grades for the FlowManager
+
+fluid.defaults("gpii.flowManager.local", {
+    components: {
+        deviceReporterDataSource: {
+            type: "kettle.dataSource"
+        },
+        journal: {
+            type: "gpii.journal",
+            options: {
+                gradeNames: ["gpii.journalLifecycleManager", "gpii.journalApp"]
+            }
+        }
     },
     requestHandlers: {
         userLogin: {
             route: "/user/:userToken/login",
             method: "get",
-            type: "gpii.flowManager.userLogin.handler"
+            type: "gpii.flowManager.userLogin.handler",
+            gradeNames: "gpii.flowManager.userLogonStateChange.matchMakingStateChangeHandler"
         },
         userLogout: {
             route: "/user/:userToken/logout",
             method: "get",
-            type: "gpii.flowManager.userLogout.handler"
+            type: "gpii.flowManager.userLogout.handler",
```

Comment:
OK, looks like we need this rule added to our ESLint - http://eslint.org/docs/rules/comma-dangle
I'll add it here and we should update Infusion etc. too. 
Looks like we can and should make a "shareable config" as per this feature: http://eslint.org/docs/developer-guide/shareable-configs